src/main/java/org/prnhs/javaee/swim/SwimApplication.java

```diff
@@ -0,0 +1,12 @@
+package org.prnhs.javaee.swim;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SwimApplication {
```

Comment:
the main app entry point.  It's designated as spring boot application, where boot will do all the setup for us.